db/migrate/20230404141831_create_planes_reservations.rb

```diff
@@ -1,6 +1,6 @@
-class CreateReservations < ActiveRecord::Migration[7.0]
+class CreatePlanesReservations < ActiveRecord::Migration[7.0]
   def change
-    create_table :reservations do |t|
+    create_table :planes_reservations do |t|
       t.date :date
       t.time :duration
```

Comment:
Optional:
There are two places we used `t.time`, but `t.time` means the time of the day. So 
the two places that we used: `life_span in planes` and here. So if possible it should be like: `interval or integer or other type of data`. If you think this change is nesseary, make a new issue for this. Work on that.